va/wayland/Makefile.am

```diff
@@ -65,5 +65,8 @@ EXTRA_DIST = \
 	wayland-drm.xml         \
 	$(NULL)
 
+BUILT_SOURCES = $(protocol_source_h)
+CLEANFILES = $(BUILT_SOURCES)
+
```

Comment:
I didn't want to include that here because the configure checks seemed a little non-standard  (like you can say "--enable-wayland" without having wayland-client installed and configure succeeds but wayland is disabled). But I can make configure fail if scanner is not available when $USE_WAYLAND.